data/src/main/java/com/github/abdallahabdelfattah13/data/repositories/dao/FeedDao.kt

```diff
@@ -0,0 +1,23 @@
+package com.github.abdallahabdelfattah13.data.repositories.dao
+
+import androidx.room.*
+import com.github.abdallahabdelfattah13.data.repositories.entities.FeedEntity
+import io.reactivex.Completable
+import io.reactivex.Flowable
+
+@Dao
+interface FeedDao {
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    fun insertFeed(feedEntity: FeedEntity): Completable
+
+    @Query("SELECT * FROM feed")
+    fun selectFeeds(): Flowable<List<FeedEntity>>
+
+    /*@Delete
+    fun deleteFeed(feedDb: FeedEntity): Completable*/
```

Comment:
If it's not bein used, you can delete this method

Reply:
I cleaned up the code.